napalm-automation/napalm-yang
# Dbarrosop/poc

DON'T MERGE, THIS IS STILL IN PROGRESS

To test/see how it works:

1. Clone this repo
2. Change to this branch
3. Execute:

```
make clean
make submodule
make models
make tests
```

Check README and `test.py` for more info.
